src/main/java/net/quantumfusion/dashloader/DashLoader.java

```diff
@@ -448,7 +448,7 @@ private void initSerializers() {
         ThreadHelper.exec(runnables);
         LOGGER.info("[3/4] [" + Duration.between(start, Instant.now()).toMillis() + "ms] Created Serializers.");
     }
-
+// le crabe 🦀
```

Comment:
The whitespace should probably be changed to be a bit more consistent.
```suggestion
    
    // le crabe 🦀
    
```